files/monitor_runs.py

```diff
@@ -134,9 +135,10 @@ def get_streaming_config(config_file, project, applet, workflow, script, token):
         config["workflow"] = workflow
     if script:
         config["script"] = os.path.abspath(script)
-    user_config_dict = yaml.load(config_file)
 
-    for key, default in CONFIG_DEFAULT.items():
+    user_config_dict = yaml.load(config_file, Loader=yaml.FullLoader)
```

Comment:
what is FullLoader needed for?

Reply:
yaml.load() without specified loader is deprecated https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation . But looking at the page right now, they say FullLoader should be avoided for now.

Reply:
I though the correct way to open yaml file is something like this:

```
with open("path") as yaml_file:
    p_dict = yaml.safe_load(yaml_file)
```

Reply:
thanks, changed